packages/expo-cli/src/commands/eject/Eject.ts

```diff
@@ -180,18 +202,20 @@ async function ejectToBareAsync(projectRoot, options) {
   await fse.writeFile(path.resolve('app.json'), JSON.stringify(appJson, null, 2));
   log(chalk.green('Wrote to app.json, please update it manually in the future.'));
 
-  let defaultDependencies = {};
+  let defaultDependencies: StringObject = {};
 
   /**
    * Extract the template and copy it over
    */
   try {
-    let tempDir = temporary.directory();
+    const tempDir = temporary.directory();
     await Exp.extractTemplateAppAsync(templateSpec, tempDir, appJson);
     fse.copySync(path.join(tempDir, 'ios'), path.join(projectRoot, 'ios'));
     fse.copySync(path.join(tempDir, 'android'), path.join(projectRoot, 'android'));
-    let packageJson = fse.readJsonSync(path.join(tempDir, 'package.json'));
-    defaultDependencies = packageJson.dependencies;
+    const { dependencies } = JsonFile.read(path.join(tempDir, 'package.json'), { json5: true });
+    if (isStringObject(dependencies)) {
```

Comment:
Not sure about this – if the template has e.g. `{"dependencies": {"lodash": 4.0, "react-native": "0.61.2"} }` (an error, because the versions should be strings), we just silently ignore the dependencies.

It seems to me that we should at least print a warning if we ignore the dependencies from the template, since this means that the template has an error that needs to be fixed.